### DISCUSSION TITLE
Any way to use an object lookup rather than CSS Vars?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [stitchesjs/stitches#491] [Help] [answered]
I'd like to use stitches, but one of the goals of a UI kit I'm creating is to have IE11 support. Is there any way to do theming without CSS Vars within Stitches?

**Comment by lifeiscontent** [comment]
@jonathantneal any guidance here would be much appreciated!

**Reply by jonathantneal** [reply]
Stitches can generate IE11 compatible CSS. However, Stitches itself is not compatible with IE11.

Therefore, I’m unsure how helpful this will be for you, but here you are — you may reference the original value from tokens like so:

```js
const { styled, theme } = createCss({
  colors: {
    red: "#ff6347"
  }
})

/* then, later... */

styled('button', {
  // this will use the original value, and not the custom property
  color: theme.colors.red.value
})
```

**Reply by lifeiscontent** [reply]
@jonathantneal hmm the theme and styled interface seem highly coupled together, how would someone be able to pass in another theme not created by me (UI kit author)? Also, what is stopping stitches from having IE11 support?

**Comment by peduarte** [comment, marked as answer]
You can try the suggestion from Jonathan, aside from that, IE11 support isnt part of the Stitches scope, sorry 🙁
